### ISSUE TITLE
[topsimulator] documentation is ancient/non-existent (Trac #1156)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1150]
Documentation in topsimulator are not reachable from the main documentation page. It should be moved to an rst file.

Most features are not documented at all. For example, the Corsika injector has many undocumented features. There are two injectors and they replicate functionality. The validation code is not documented. No test/example is documented. The corsika reader is not documented.

Migrated from https://code.icecube.wisc.edu/ticket/1156
```json
{
    "status": "closed", 
    "changetime": "2016-03-18T21:14:03", 
    "description": "Documentation in topsimulator are not reachable from the main documentation page. It should be moved to an rst file.\n\nMost features are not documented at all. For example, the Corsika injector has many undocumented features. There are two injectors and they replicate functionality. The validation code is not documented. No test/example is documented. The corsika reader is not documented.", 
    "reporter": "jgonzalez", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1458335643235016", 
    "component": "combo simulation", 
    "summary": "[topsimulator] documentation is ancient/non-existent", 
    "priority": "blocker", 
    "keywords": "", 
    "time": "2015-08-18T13:22:29", 
    "milestone": "", 
    "owner": "jgonzalez", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20150818T13:23:15: jgonzalez changed status from "new" to "accepted"

**Comment by icecube-trac** [comment]
Trac update at 20150824T18:05:29: 
* olivas changed priority from "normal" to "blocker"
* olivas changed component from "cmake" to "simulation"

**Comment by icecube-trac** [comment]
Trac update at 20150917T22:36:32: 
* hdembinski changed status from "accepted" to "closed"
* hdembinski changed resolution from "" to "fixed"
* hdembinski commented:

> In changeset 137713:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="137713"
> standardize and update docu, closes #1150
> ```

**Comment by icecube-trac** [comment]
Trac update at 20160318T21:14:03: 
* olivas changed milestone from "IceSim 4.2" to ""
* olivas commented:

> Milestone IceSim 4.2 deleted
